source/_integrations/mastodon.markdown

```diff
@@ -17,7 +17,12 @@ The `mastodon` platform uses [Mastodon](https://joinmastodon.org/) to deliver no
 
 ### Setup
 
-Go to **Preferences** in the Mastodon web interface, then to **Development** and create a new application.
+Go to **Preferences** in the Mastodon web interface, then to **Development** and create a new application. This
+integration currently only support sending notifications, so the API only has to be able to `write` to the
+account.
+
```

Comment:
Do you think you could maybe also specify what to put for **Application website** and **Redirect URI** on the Mastodon end of things? If nothing is required there, maybe just say that it's fine to leave those fields empty. I tried following your steps and felt uncertain there.